modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/rest-openapi.yaml

```diff
@@ -52,6 +52,8 @@ components:
                     format: date
                     readOnly: true
                     type: string
+                externalReferenceCode:
```

Comment:
Rename this commit to "Add external reference code property to ListTypeEntry in REST layer"